static/js/popovers.js

```diff
@@ -390,6 +392,8 @@ export function show_user_profile(user) {
         user_time: people.get_user_time(user.user_id),
         user_type: people.get_user_type(user.user_id),
         user_is_guest: user.is_guest,
+        groups_of_user,
```

Comment:
I think this should be, part of next commit as that solely deals with using these functions to display the information given by them on template.